docs/src/main/sphinx/release/release-469.md

```diff
@@ -1,89 +1,175 @@
-# Release xyz (dd MMM 2024)
+# Release 469 (16 Jan 2025)
 
 ## General
 
-## Security
+* Add geospatial constructor {func}`ST_GeomFromKML`. ({issue}`24297`)
+* Allow deactivating the spooling protocol with the `spooling_protocol_enabled`
+  session property. ({issue}`24655`)
+* Improve stability of the cluster under load. ({issue}`24572`) 
+* Refrain from join pushdown for modified tables. ({issue}`24447`)
+* Fix parsing of negative `0x`, `0b`, `0o` long literals. ({issue}`24601`)
 
 ## Web UI
 
-## JDBC driver
+* Add support for filtering by query client tags. ({issue}`24494`)
+* Add authentication support for the Preview Web UI with the authentication
+  types `FORM`, `FIXED`, `OAUTH2`, `JWT`, `KERBEROS`, `CERTIFICATE`, and
+  `INSECURE`. ({issue}`23230`)
+* Sort queries on the current worker status section in the Preview Web UI by
+  descending reserved memory. ({issue}`24475`)
 
-## Server RPM
+## JDBC driver
 
-## Docker image
+* Add `planningTimeMillis`, `analysisTimeMillis`, `finishingTimeMillis`,
+  `physicalInputBytes`, `physicalWrittenBytes`, `internalNetworkInputBytes` and
+  `physicalInputTimeMillis` to `io.trino.jdbc.QueryStats`. ({issue}`24571`,
+  {issue}`24604`)
+* Add the `validateConnection` connection property to allows validating
+  connection, and improve `Connection.isValid(int)` so it validates connection
+  and credentials. ({issue}`24127`, {issue}`22684`)
+* Fix loading spooled segment data when cluster is secured. ({issue}`24595`)
```

Comment:
I don’t understand this one

Reply:
This is about http client truststore configuration in regard to spooling protocol

Reply:
Reworded